docs/static/options.yaml

```diff
@@ -1,5 +1,6 @@
 defaults:
   - architecture: soap_bpnn  # architecture used to train the model
+  - _self_
 
 # Section defining the parameters for structure and target data
 dataset:
```

Comment:
this looks quite a bit different from the previous discussion, is this the old version of the parameters?

Reply:
Yes it still builds on the current main branch, where the new option was not in until 5 minutes ago.


I will rebase and update the config.